src/main/java/taxi/service/AuthenticationService.java

```diff
@@ -0,0 +1,9 @@
+package taxi.service;
+
+import taxi.exception.AuthenticationException;
+import taxi.model.Driver;
+
+public interface AuthenticationService {
+

```

Comment:
empty line

Reply:
please, remove all redudnant lines in your pr

Reply:
fixed))